lang/en.json

```
@@ -208,52 +208,52 @@
-        "ErrorMessageVariableDescription": "Returns the message of an error, when an error was triggered previously.",
+        "ErrorMessageVariableDescription": "Returns the message of an error, when an error was previously triggered.",
         "ErrorMessageVariableName": "Error Message",
         "ErrorNameDuplicate": "This name is already in use.",
         "ErrorNotAnArray": "\"%1$s\" has to be an array.",
         "ErrorPreviewReservedEnvironment": "The preview environment cannot be configured as it is a reserved environment",
         "ErrorTriggerAtPositionXDoesNotExist": "The trigger \"%1$s\" at position \"%2$s\" does not exist.",
-        "ErrorTriggerNotRemovableAsInUse": "This trigger cannot be deleted as it is used in a tag. To remove this trigger, update any referenced tag first.",
-        "ErrorUrlVariableDescription": "Returns the URL of an error, when an error was triggered previously.",
-        "ErrorUrlVariableName": "Error Url",
-        "ErrorVariableInConditionAtPositionNotFound": "The variable \"%1$s\" in the condition at position \"%2$s\" cannot be found.",
+        "ErrorTriggerNotRemovableAsInUse": "This trigger cannot be deleted as it is used in a tag. Remove this trigger by updating any referenced tag first.",
+        "ErrorUrlVariableDescription": "Returns the URL of an error, when an error was previously triggered.",
+        "ErrorUrlVariableName": "Error URL",
+        "ErrorVariableInConditionAtPositionNotFound": "Could not find the variable \"%1$s\" in the condition at position \"%2$s\".",
         "ErrorVariableInvalidDefaultValue": "The default value needs to be empty, a string, or a number.",
-        "ErrorVariableNameInUseByPreconfiguredVariable": "This variable name cannot be used as a preconfigured variable already uses this name.",
+        "ErrorVariableNameInUseByPreconfiguredVariable": "This variable name cannot be used, as a preconfigured variable already uses this name.",
         "ErrorVersionCannotBeDeleted": "The \"%1$s\" version of a container cannot be deleted.",
         "ErrorVersionCannotBeDeletedAsPublished": "This version cannot be deleted as it is published to at least one environment.",
         "ErrorXNotProvided": "Please provide a value for \"%1$s\".",
         "ErrorXNotWhitelisted": "The value for \"%1$s\" is not allowed, use one of: %2$s.",
-        "ErrorXTooLong": "\"%1$s\" is too long, max %2$s characters are allowed.",
+        "ErrorXTooLong": "\"%1$s\" is too long, at most %2$s characters are allowed.",
         "EtrackerTagName": "etracker web analysis",
         "EtrackerTagDescription": "etracker is a web analytics system, you can choose to send a pageview or an event.",
         "EtrackerTagHelp": "",
         "Except": "Except",
         "ExportDraft": "Export draft",
         "ExportX": "Export %s",
         "FacebookPixelTagDescription": "The Facebook Pixel is a web analytics and advertising service offered by Facebook.",
```

```suggestion
        "FacebookPixelTagDescription": "The Facebook Pixel is a web analytics and advertising service offered by Meta.",
```
? or "A first-party …" to avoid it all-together?

---

Meta should be fine.